src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpoint.java

```diff
@@ -460,6 +460,33 @@ private void validateConstraints(final CollectionExerciseDTO collexDto) throws C
 
     }
 
+    /**
+     * PUT request for unlinking sample summary from a collection exercise
```

Comment:
Maybe just remove the PUT and change to for unlinking sample summary from a collection exercise